.github/workflows/branches.yml

```diff
@@ -0,0 +1,17 @@
+name: Build
+
+on:
+  push:
+    branches-ignore:
+      - master
```

Comment:
Disse kan vel ignoreres?

Reply:
Jeg gjorde det sånn fordi en push til master blir allerede bygget av `master.yml` workflowen. Det virker unødvendig å fyre av to bygg av master, når det er såpass enkelt å sørge for å ikke gjøre det. :slightly_smiling_face:

Reply:
Hvis workflowen ikke trigges på `push`, men bare på `pull_request` vil ikke workflowen kjøres på master, og da trenger man ikke ignorere master heller.

Reply:
Det er sant. Men det er forskjell på bygg trigget av `push` og bygg trigget av `pull_request`:

`push`: Bygger koden på denne commiten i denne branchen
`pull_request`: Bygger koden slik den vil se ut hvis du merger denne branchen inn i target branch

Jeg syns det er greit å ha begge, men åpen for å droppe push altså.